CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(APPLE)
 	set_property(DIRECTORY
 		APPEND
 		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
```

Comment:
@scztt, sorry for extending this thread a bit: running sc in Debug mode from the IDE doesn't seem exactly self-explanatory. Is it the best way to "build" the "SuperCollider" Scheme rather than the all_builds target?